### ISSUE TITLE
No break opportunity between math or svg

### ISSUE THREAD

**Issue by MurakamiShinyu** [vivliostyle/vivliostyle.js#750]
**Example 1:**

```html
<math display="block">
  …
</math>
<math display="block">
  …
</math>
```

JSBin: https://jsbin.com/giqulef

Test with Vivliostyle Viewer: https://vivliostyle.vercel.app/#src=https://jsbin.com/giqulef

**Example 2:**

```html
<p data-math-typeset="true">
  When \(a \ne 0\), there are two solutions to \(ax^2 + bx + c = 0\) and
  they are $$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$
</p>
<p data-math-typeset="true">
  When \(a \ne 0\), there are two solutions to \(ax^2 + bx + c = 0\) and
  they are $$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$
</p>
```

JSBin: https://jsbin.com/soheduf

Test with Vivliostyle Viewer: https://vivliostyle.vercel.app/#src=https://jsbin.com/soheduf

**Comment by MurakamiShinyu** [comment]
**Same problem on SVG**

JSBin: https://jsbin.com/tisanib
Test with Vivliostyle Viewer: https://vivliostyle.vercel.app/#src=https://jsbin.com/tisanib

**Comment by MurakamiShinyu** [comment]
Fixed in #788 

Test:
- https://vivliostyle-git-fix-issue750-vivliostyle.vercel.app/#src=https://jsbin.com/giqulef
- https://vivliostyle-git-fix-issue750-vivliostyle.vercel.app/#src=https://jsbin.com/soheduf
- https://vivliostyle-git-fix-issue750-vivliostyle.vercel.app/#src=https://jsbin.com/tisanib
